hypershift-operator/controllers/nodepool/nodepool_controller.go

```diff
@@ -1357,9 +1412,10 @@ func (r *NodePoolReconciler) getConfig(ctx context.Context,
 
 func (r *NodePoolReconciler) getTunedConfig(ctx context.Context,
 	nodePool *hyperv1.NodePool,
-) (configsRaw string, err error) {
+) tunedConfigData {
```

Comment:
Related to my previous comment, I think we could get rid of the tunedConfigData struct by returning
```suggestion
) (tunedConfigsRaw, performanceProfileConfigsRaw, error) {
```